keps/sig-cluster-lifecycle/kubeadm/2568-kubeadm-non-root-control-plane/README.md

```diff
@@ -617,6 +619,15 @@ In addition to the file/directories in that table above the control-plane compon
 - /etc/ssl/certs
 - /etc/ca-certificates
 
+### Reusing users and groups
+
+If any of the users/groups defined above exist already and are in the expected ID range of `SYS_UID_MIN`-`SYS_UID_MAX` for users and `SYS_GID_MIN`-`SYS_GID_MAX` for groups, then `kubeadm` will reuse these instead of creating a new ones. More specifically `kubeadm` will reuse the ones that exist and meet the criteria and will create the ones that it needs.
```

Comment:
```suggestion
If any of the users/groups defined above exist already and are in the expected ID range of `SYS_UID_MIN`-`SYS_UID_MAX` for users and `SYS_GID_MIN`-`SYS_GID_MAX` for groups, then `kubeadm` will reuse these instead of creating new ones. More specifically `kubeadm` will reuse the ones that exist and meet the criteria and will create the ones that it needs.
```